samples/meetings-stage-view/nodejs/ClientApp/package.json

```diff
@@ -5,18 +5,17 @@
   "author": "Microsoft",
```

Comment:
Why we remove ConfigController  & package-lock.json from both C# and nodejs folder ?